recipes/cuda-kat/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+class CudaKatConan(ConanFile):
+    name = "cuda-kat"
+    homepage = "https://github.com/eyalroz/cuda-kat"
+    description = "CUDA kernel author's tools"
+    topics = ("GPGPU", "CUDA", "cuda-kat", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    no_copy_source = True
+    license = "BSD 3-Clause License"
+
+    settings = "os"
```

Comment:
```suggestion
    settings = "os", "arch", "compiler", "build_type"
```
Keep full settings here, for any recipe. It will be default behavior for Conan 2.0